integration-tests/utils/integration-utils.js

```diff
@@ -97,7 +97,7 @@ const assertRowHas = async (page, { tableName, expectedColumnValues, withKey: {
 
 const clickTableCell = async (page, { tableName, columnHeader, textContains, isDescendant = false }, options) => {
   const tableCellPath = await getTableCellByContents(page, { tableName, columnHeader, textContains, isDescendant })
-  const xpath = `${tableCellPath}//*[@role="button" or @role="link" or @role="checkbox"]`
+  const xpath = `${tableCellPath}[@role="button" or @role="link" or @role="checkbox"]`
```

Comment:
Could you explain the reasoning behind the change above?

Reply:
has to change the test in `catalog-utils.js` to something that exist and clickable in both dev and alpha, so tested for 'Readable Catalog Snapshot 1'. and was discovered that the name was on the same path level as @role="button"